File: C++/프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> num_list) {
    int answer = 0;
    int result1 = 1;
    int result2 = 0;
    for (int i=0; i < num_list.size(); i++){
        result1 *= num_list[i];
        result2 += num_list[i];
    }
    if (result2*result2 > result1) {
        answer = 1;
    }
    return answer;
}

File: JavaScript/프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.js
function solution(num_list) {
    let answer = 0;
    let result1 = 1;
    let result2 = 0;
    
    for(let i=0; i<num_list.length; i++){
        result1 *= num_list[i];
        result2 += num_list[i];
    }
    
    if(result1 < result2*result2) { answer = 1; }
    return answer;
}